### DISCUSSION TITLE
Different intensity range in Sepia (STISuite v3.0 ) compared with STISuite v3.0

### DISCUSSION THREAD

**Discussion by zoelir729** [kschan0214/sepia#25] [General]
nifti file: https://wwa.lanzoui.com/iNiunsg5wef
Question1: 

Mask file was created by FSL brain extration (bet) in Sepia.

L: in Sepia (STISuite v3.0 ), mask file =Sepia_mask.nii.gz

R: in STISuite v3.0，mask file = Sepia_mask.nii.gz

**Intensity Range was different: L (-0.1792:0.2911), R (-0.2003:0.3273)**

![image](https://user-images.githubusercontent.com/36666968/128829067-af9258f4-f9d0-4831-b825-9338170a952c.png)
![image](https://user-images.githubusercontent.com/36666968/128829138-8086ea31-7f62-4085-aa40-412c8752d2c2.png)

Question2:

Mask file was created by `"single button for QSM!"` in STISuite v3.0.

This mask is different from mask created by "bet", seemed to be better for QSM reconstruction according to the QSM file which showed the internal pallidum structure.

L: in Sepia (STISuite v3.0 ), mask file =S001_Mask_STISuite_singlebutton.nii

R: in STISuite v3.0，mask file = S001_Mask_STISuite_singlebutton.nii

**Intensity Range was different: L (-0.4812:0.376), R (-0.4722:0.4511)**

![image](https://user-images.githubusercontent.com/36666968/128829205-e8e6a0b4-d191-41ce-bdc9-5967f3adeb3c.png)
![image](https://user-images.githubusercontent.com/36666968/128829270-6beb2715-d5e1-43c5-93ba-7fe01f568031.png)

[Q_Different_intensity_range_in_Sepia_(STISuite_v3.0_)_compared_with_STISuite_v3.0.pdf](https://github.com/kschan0214/sepia/files/6959879/Q_Different_intensity_range_in_Sepia_.STISuite_v3.0_._compared_with_STISuite_v3.0.pdf)

**Comment by kschan0214** [comment]
Hi Zoe,

Regrading your first question, the differences are likely due to the slightly different pipeline implementation in SEPIA compared to the template pipeline of STI suite. This is because we need to accommodate a wider range of scenarios in SEPIA. If you are interested in reproducing the same result on both SEPIA and STI Suite, you could check on the script attachment in https://github.com/kschan0214/sepia/discussions/23#discussioncomment-1128873. 

For your second question, the basal ganglia shows better appearance is because of the wider display window (~ +/-0.5, bottom images). Perhaps if you put the same display window on the results shown in question 1, you would be able to see similar contrast. The reason of having a wider display range in the second results is likely to be caused by the inaccurate brain mask in the frontal region, resulting in inaccurately wide dynamic range of the magnetic susceptibility (and artefacts) in the frontal regions.

P.S. I'll convert this issue to a discussion topic, as this is more related to the implementation differences of the QSM processing pipeline.
